### PR TITLE
DOCS-6091 Migrating CI Visibility Guide Content into Documentation

### PR DESCRIPTION
**Pull request by alai97**
### Motivation

DOCS-6091

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/_index.md [hunk 1]
```diff
@@ -45,6 +46,50 @@ cascade:
...
+
+In CI Visibility, a test with multiple configurations is treated as multiple tests with a separate test for each configuration. In the case where one of the configurations fails but the others pass, only that specific test and configuration combination is marked as failed.
+
+For example, suppose you're testing a single commit and you have a Python test that runs against three different Python versions. If the test fails for one of those versions, that specific test is marked as failed, while the other versions are marked as passed. If you retry the tests against the same commit and now the test for all three Python versions pass, the test with the version that previously failed is now marked as both passed and flaky, while the other two versions remain passed, with no flakiness detected.
```
Comment (alai97) [inline content/en/continuous_integration/tests/_index.md:55]:
```suggestion
For example, suppose you're testing a single commit and you have a Python test that runs against three different Python versions. If the test fails for one of those versions, that specific test is marked as failed, while the other versions are marked as passed. 

If you retry the tests against the same commit and now the test for all three Python versions pass, the test with the version that previously failed is now marked as both p assed and flaky, while the other two versions remain passed, with no flakiness detected.
```

**Review by juan-fernandez — COMMENTED** [review]
thanks for this, Austin. It's always great when we try to make the documentation better! 

❓ is the idea to move every one of the guides (Flaky test management, instrumenting your browser tests with RUM, etc.) into the root of the test page, the same way we do with Test configurations in this PR? 

If that's the case, wouldn't that root be huge and difficult to navigate? If the idea is to move only some of the sections, what criterion are we using?

File: config/_default/menus/menus.en.yaml [hunk 2]
```diff
@@ -2302,41 +2302,66 @@ main:
     parent: intelligent_test_runner
     identifier: ci_itr_swift
     weight: 506
+  - name: Connect Test Visibility
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2305]:
```suggestion
  - name: Integrate Test Visibility
```

File: content/en/continuous_integration/search/_index.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ---
 title: Search and Manage CI Tests and Pipelines 
 description: Learn how to search for your CI tests and pipelines.
+aliases:
+- /continuous_integration/guides/find_flaky_tests/
+- /continuous_integration/guides/flaky_test_management/
 further_reading:
```
Comment (alai97) [inline content/en/continuous_integration/search/_index.md:7]:
```suggestion
algolia:
   keyword: "flaky test, flaky tests"
further_reading:
```

**Review by KickingTheTV — APPROVED** [review]
LGTM

**Comment by aliciascott** [comment]
Thanks! Created Docs card for Editorial Review https://datadoghq.atlassian.net/browse/DOCS-6290

File: content/en/continuous_integration/explorer/_index.md [hunk 4]
```diff
@@ -27,25 +27,25 @@ Navigate to [**CI** > **Test Runs**][101] to see your CI test run results across
...
+| Facet | Description |
+|---|---|
+| Test Status | The test status: `Passed` or `Failed`. |
+| Duration | Length of time for the test to complete. |
+| Test Service | The test service instrumented with CI Visibility. |
+| Test Name | The name of the test. |
+| Test Suite | The name of the service. |
+| Flaky | Whether or not a test is a flaky test: `true` or `false`, meaning that the test run has flaked in the commit.<br><br>Either this test run has failed and there was a previous test run for the same test and commit that passed, or this test run passed and there is a previous failure. This attribute is only present the first time a test is detected flaky in a commit. |
+| Has Parameters | Whether or not a test has parameters: `true` or `false`. |
+| Known Flaky | Whether or not a test is known to be flaky: `true` or `false`. <br><br>The test run has flaked in the commit (according to the “Flaky” definition) and it was already present on the “Flaky test” table of either the branch of the test run or the default branch of the repository. This means that the test was detected as flaky in the past, and that this failure may not be the result of a change in this commit. |
```
Comment (drichards-87) [inline content/en/continuous_integration/explorer/_index.md:39]:
```suggestion
| Known Flaky | Whether or not a test is known to be flaky: `true` or `false`. <br><br>The test run has flaked in the commit (according to the "Flaky" definition) and it was already present on the "Flaky test" table of either the branch of the test run or the default branch of the repository. This means that the test was detected as flaky in the past, and that this failure may not be the result of a change in this commit. |
```

File: content/en/continuous_integration/explorer/_index.md [hunk 5]
```diff
@@ -27,25 +27,25 @@ Navigate to [**CI** > **Test Runs**][101] to see your CI test run results across
...
+| Language | The language of the service. |
+| New Flaky | Whether or not this flaky test has occurred before: `true` or `false`. <br><br>The test run has flaked in the commit (according to the “Flaky” description) and didn’t previously exist in the **Flaky Tests** table for the branch of the test run or default branch of the repository. This also means that the test has been added to the **Flaky Tests** table. |
```
Comment (drichards-87) [inline content/en/continuous_integration/explorer/_index.md:41]:
```suggestion
| New Flaky | Whether or not this flaky test has occurred before: `true` or `false`. <br><br>The test run has flaked in the commit (according to the "Flaky" description) and didn’t previously exist in the **Flaky Tests** table for the branch of the test run or default branch of the repository. This also means that the test has been added to the **Flaky Tests** table. |
```

**Review by drichards-87 — APPROVED** [review]
Looks good! I just had the two suggestions for changing the smart quotes to straight quotes. PR is approved.

**Comment by fermayo** [comment]
nvm, the guide list just updated for me. Could have been my local browser cache

**Pull request merged**